docs/docsite/rst/command_guide/cheatsheet.rst

```diff
@@ -105,3 +105,27 @@ Runs  ``ansible localhost``- on your local system.
     "cache_updated": false,
     "changed": false
     #...
+
+ansible doc
+===========
+
+* Show plugin names and their source files:
+
+.. code-block:: bash
+
+  ansible-doc -F
+  add_host                                                                       /usr/lib/python3/dist-packages/ansible/modules/add_host.py   
+  amazon.aws.aws_az_facts                                                        /usr/lib/python3/dist-packages/ansible_collections/amazon/aws/plugins/modules/aws_az_f
+  amazon.aws.aws_az_info                                                         /usr/lib/python3/dist-packages/ansible_collections/amazon/aws/plugins/modules/aws_az_i
+  amazon.aws.aws_caller_facts                                                     /usr/lib/python3/dist-packages/ansible_collections/amazon/aws/plugins/modules/aws_call
```

Comment:
What do you think about just listing the command with the option and not including the sample output?

As this is a cheatsheet I think it's better to be a bit less verbose.

Reply:
Yeah,I was also thinking the same.That will look compact too.Thanks @oraNod